examples/python-json-datagenerator/predefinedschema.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import json
+import random
+
+model = "llama2"
+template = {
+  "firstName": "", 
+  "lastName": "", 
+  "address": {
+    "theStreet": "", 
+    "theCity": "", 
+    "theState": "", 
+    "theZipCode": ""
```

Comment:
Just a suggestion
```suggestion
    "street": "", 
    "city": "", 
    "state": "", 
    "zipCode": ""
```

Reply:
the point was to choose a schema that the model would never think of. To prove that it was defined by me.

Reply:
Yes best to stick to a schema folks are likely to use / need!